build.gradle

```diff
@@ -81,8 +81,8 @@ subprojects {
 
     [compileJava, compileTestJava]*.options*.compilerArgs = ["-Xlint:none", "-nowarn"]
 
```

Comment:
Can we have the constants from https://github.com/cloudfoundry/uaa/pull/2552/files ?
@peterhaochen47 we can merge https://github.com/cloudfoundry/uaa/pull/2552 or you can adopt, however @klaus-sap mention inconsistencies... I think JavaVersion.VERSION_11 is 11 simply and not 1.11, however if we use always constants it is solved

Reply:
JavaVersion.VERSION_11 is '1.11' (with quotation marks). Setting sourceCompatibility = 1.11 (w/o quotation marks) does not work on Mac

Reply:
Hi, just made a commit about this.